pkg/client/listers.go

```diff
@@ -33,3 +33,14 @@ type Listers struct {
 	ProxyConfigs        configlisters.ProxyLister
 	Infrastructures     configlisters.InfrastructureLister
 }
+
+type ImagePrunerControllerListers struct {
+	Jobs                kjoblisters.JobNamespaceLister
+	CronJobs            kbatchlisters.CronJobNamespaceLister
+	ServiceAccounts     kcorelisters.ServiceAccountNamespaceLister
+	Routes              routelisters.RouteNamespaceLister
```

Comment:
This lister is not used by the pruner controller.

Reply:
thanks, i missed that one.  i think i want to refactor this to share the listers between controllers in a followup pr.